scantools/run_navvis_to_capture.py

```diff
@@ -134,6 +133,55 @@ def run(input_path: Path, capture: Capture, tiles_format: str, session_id: Optio
                     pose = get_pose(nv, upright, frame_id, camera_id, tile_id)
                     trajectory[timestamp_us, sensor_id] = pose
 
+    if export_trace and export_as_rig:
+        # Add "trace" to the rig with identity pose.
+        rigs[rig_id, "trace"] = Pose()
+
+        # Add "trace" as a sensor.
+        sensors['trace'] = create_sensor('trace', name='Mapping path')
+
+        qvec, tvec = nv._imu["orientation"], nv._imu["position"]
+        camhead_from_imu = Pose(r=qvec, t=tvec)
```

Comment:
not this PR: given that now we actually parse and store the cam-head, we could probably update the rig-frame to be the same as camhead (and thus maybe avoiding the the cam0 origin additional logic)?

Reply:
Yes, I want to do this. It will be a cleaner option. I will do it in another PR. Thanks

Reply:
https://github.com/microsoft/lamar-benchmark/issues/64